src/metabase/models/permissions.clj

```diff
@@ -418,20 +428,7 @@
        (when config/is-test?
          (throw e))))))
 
-; Audit Permissions helper fns
-
-(defn check-audit-db-permissions
-  "Check that the changes coming in does not attempt to change audit database permission. Admins should
-  change these permissions in application monitoring permissions."
-  [changes]
-  (let [changes-ids (->> changes
-                         vals
-                         (map keys)
-                         (apply concat))]
-    (when (some #{audit/audit-db-id} changes-ids)
-      (throw (ex-info (tru
-                       (str "Audit database permissions can only be changed by updating audit collection permissions."))
-                      {:status-code 400})))))
```

Comment:
LSP informed me that this function was not actually used anywhere, so I went ahead and removed it. There is a function with this name in `metabase-enterprise.audit-app.permissions` that is still being used